src/bundle/lookup.js

```diff
@@ -0,0 +1,42 @@
+define([
+	"../core/likely_subtags",
+	"../core/remove_likely_subtags",
+	"../core/subtags",
+	"../util/array/for_each"
+], function( coreLikelySubtags, coreRemoveLikelySubtags, coreSubtags, arrayForEach ) {
+
+	/**
+	 * bundleLookup( minLanguageId )
+	 *
+	 * @Cldr [Cldr class]
+	 *
+	 * @cldr [Cldr instance]
+	 *
+	 * @minLanguageId [String] requested languageId after applied remove likely subtags.
+	 *
+	 * Ref: http://goo.gl/qpwZ4n
```

Comment:
This document might change, get deleted, whatever. With the documentation in place, is it really needed?